### PR TITLE
Add MySQL migration test.

### PR DESCRIPTION
**Pull request by amit-yb**

### CODE REVIEW

File: installer_scripts/common.sh [hunk 1]
```diff
@@ -36,8 +36,10 @@ update_yb_migrate_bashrc() {
 
 install_yb_migrate() {
 	output "Installing yb_migrate."
-	$GO install github.com/yugabyte/yb-db-migration/yb_migrate@latest
+	pushd ../yb_migrate
+	$GO install
```
Comment (sanyamsinghal) [inline installer_scripts/common.sh:40]: any specific reason for this change? Both ways are fine, right?

File: migtests/scripts/functions.sh [hunk 2]
```diff
@@ -92,7 +98,7 @@ import_schema() {
 }
 
 import_data() {
-	yb_migrate import data --export-dir ${EXPORT_DIR} \
+	yes | yb_migrate import data --export-dir ${EXPORT_DIR} \
```
Comment (sanyamsinghal) [inline migtests/scripts/functions.sh:101]: where do we require yes in import data also?

**Review by sanyamsinghal — APPROVED** [review]
LGTM! Left some minor comments, please respond before merging.

File: installer_scripts/common.sh (hunk 1 above)
Comment (amit-yb) [reply installer_scripts/common.sh:40]: With the earlier command, the pre-commit tests were always installing `yb_migrate` from the `main` branch. What we actually need is to test the migration with the changes in the PR.

File: migtests/scripts/functions.sh (hunk 2 above)
Comment (amit-yb) [reply migtests/scripts/functions.sh:101]: For MySQL, when we try to import in the `public` schema, yb_migrate prompts for confirmation.

Comment (sanyamsinghal) [reply migtests/scripts/functions.sh:101]: ok, i thought it is only for import schema.

File: installer_scripts/common.sh (hunk 1 above)
Comment (sanyamsinghal) [reply installer_scripts/common.sh:40]: Thanks.

**Pull request merged**